### PR TITLE
Escape comments

### PR DESCRIPTION
**Pull request by SimonSiefke**
Escape comments like /* */ in description properties

### CODE REVIEW

File: src/normalizer.ts [hunk 1]
```diff
@@ -43,6 +43,30 @@ rules.set('Default top level `id`', (schema, rootSchema, fileName) => {
...
+  replacer = '*\\/'
+) {
+  if (code === null || typeof code !== 'object') {
+    return
+  }
+  for (const key of Object.keys(code)) {
+    if (key === 'description' && typeof code[key] === 'string') {
+      code[key] = code[key].replace(/\*\//g, replacer)
+    } else {
+      escape(code[key])
```
Comment (bcherny) [inline src/normalizer.ts]: I don't think you need to manually recurse. IIRC `mapDeep` handles this for you.

File: src/normalizer.ts [hunk 2]
```diff
@@ -43,6 +43,30 @@ rules.set('Default top level `id`', (schema, rootSchema, fileName) => {
   return schema
 })
 
+function escape(
+  code: string | { [key: string]: any },
```
Comment (bcherny) [inline src/normalizer.ts]: Call this `schema`, to align with the other rules. Also, its type should just be `JSONSchema`.

File: src/normalizer.ts [hunk 3]
```diff
@@ -43,6 +43,30 @@ rules.set('Default top level `id`', (schema, rootSchema, fileName) => {
   return schema
 })
 
+function escape(
```
Comment (bcherny) [inline src/normalizer.ts]: Call this `escapeBlockComment`, and move it to *utils.ts*.

**Comment by SimonSiefke** [comment]
Can you take another look? :)

**Comment by bcherny** [comment]
Published `6.1.2`

**Pull request merged**